com.unity.shadergraph/Editor/Data/Graphs/VirtualTextureShaderProperty.cs

```diff
@@ -184,11 +184,13 @@ internal void AddTextureInfo(List<PropertyCollector.TextureInfo> infos)
             {
                 string layerRefName = value.layers[layer].layerRefName;
                 var layerTexture = value.layers[layer].layerTexture;
+                var texture = layerTexture != null ? layerTexture.texture : null;
```

Comment:
FYI if you didn't know about the fancy C# syntax that does the same thing:

var texture = layerTexture?.texture;

Reply:
Thanks! I was aware of that syntax, but I try to avoid it with Object derived types. Not the case here, I'll change it.